package.json

```diff
@@ -47,9 +47,9 @@
     "@babel/preset-react": "7.10.1",
     "@parse/minami": "1.0.0",
     "@saithodev/semantic-release-backmerge": "2.1.3",
-    "@semantic-release/changelog": "5.0.1",
```

Comment:
Could you please only do the packagelog v2 upgrade in this PR and remove all other upgrades? We want to look at these things separately.

Reply:
> We do dependency upgrades in separate commits for easier debugging.

This is true but cause issues. V2 is restrictive on peer dependencies.
https://github.com/parse-community/Parse-SDK-JS/actions/runs/4089591152/jobs/7052405616

You downgraded semantic-release instead of bumping its peer dependencies. Most of our packages are grouped together and outdated.

* babel-jest + jest
* eslint + flowtype + jsdocs
* babel/runtime-corejs + core js + babel/core + babel/plugins
* semantic-release

> We want to keep the workflow files conceptually coherent across repos

I'll make this change

Reply:
> Can you research it now?

Please revert a y semantic release related changes.

Reply:
> We can keep the dependency upgrades that are necessary for the lockfile upgrade, that makes sense.

The semantic-release changes are necessary for the V2 upgrade unless you downgrade semantic-release even further from 19.0.5.

Reply:
OK, let me take a quick look the semantic release versions here

Reply:
These are the versions used in Parse Server which is on lock-file v2; so I doubt that any of the semantic-release upgrades is required for this PR.

```
"@semantic-release/changelog": "5.0.1",
"@semantic-release/commit-analyzer": "8.0.1",
"@semantic-release/git": "9.0.0",
"@semantic-release/github": "7.2.3",
"@semantic-release/npm": "7.1.3",
"@semantic-release/release-notes-generator": "9.0.3",
```

Reply:
You missed the most important one
```
"semantic-release": "17.4.6",
```

Reply:
You were reading my comment faster than I was able to edit it :-) So I've looked through all the changelogs now and I guess we can merge this.